.github/workflows/release.yml

```diff
@@ -15,39 +15,35 @@ on:
     tags:
       - 'v*'
 jobs:
-  go-version:
-    runs-on: ubuntu-latest
-    outputs:
-      go-version: ${{ steps.go-version.outputs.go-version }}
-    steps:
-      - uses: actions/checkout@v2
-      # Read the .go-version file and output it for other jobs to use.
-      - id: go-version
-        run: echo "::set-output name=go-version::$(cat .go-version)"
-
   goreleaser:
     runs-on: ubuntu-latest
-    needs: [go-version]
     steps:
-      - uses: actions/setup-go@v2
-        with:
-          go-version: ${{ needs.go-version.outputs.go-version }}
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
-        with:
-          path: ~/go/pkg/mod
-          key: ${{ runner.os }}-go${{ env.GO_VERSION }}-${{ hashFiles('go.sum') }}-no-tools
-      - name: Unshallow
+      -
+        name: Checkout
+        uses: actions/checkout@v2.4.0
+      -
+        name: Unshallow
         run: git fetch --prune --unshallow
-      - uses: paultyng/ghaction-import-gpg@v2.1.0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
```

Comment:
Could you revert this line and use the go-version from the go-version job, so we use the same go-version in the release as we do in the build/test?

Reply:
Done.